services/QuillLMS/app/views/pages/research_and_efficacy.html.erb

```diff
@@ -0,0 +1,271 @@
+            loading="lazy"
+          />
+          <img
+            class="report-badge"
+            src=<%= "#{base_image_url}/leanlab-education-essa-tier-logo.png" %>
+            alt="Leanlab Education Essa Tier logo"
+            loading="lazy"
+          />
+        </figure>
+        <figcaption class="caption">Study conducted by Leanlab Education</figcaption>
+      </div>
+      <section>
+        <p>2025 Leanlab Education Study</p>
+        <h2>Quill students improved writing skills 1.6x faster, with the largest gains in Title I schools</h2>
+        <p class="body-text">The study of 100,200 students included diverse educational contexts, including urban, suburban, and rural settings. The intervention was completed by students in grades 4–12, with the majority of students in the study (88%) attending grades 6-8. Students who used Quill improved their writing skills 1.6 times faster than those who did not, and the more they practiced, the more they gained. Students in Title I schools made the biggest progress. These results meet ESSA Tier III standards and show that Quill can make a meaningful, scalable difference in sentence-level writing growth.</p>
+        <a class="quill-button medium contained green focus-on-light" href="https://s3.amazonaws.com/quill-image-uploads/uploads/files/LeanLab_Education_s_Quill.org_Efficacy_Study_-_ESSA_Tier_III_Efficacy_Report_On_The_Impact_of_Quill.org_on_Student_Language_Skill_Development_3454.pdf" rel="noopener noreferrer" target="_blank">Download Report PDF</a>
+      </section>
+    </section>
+  </section>
+  <section class="graph-section growth">
+    <section class="inner-section">
+      <section>
+        <h2 class="first">1.6x Rate of Growth</h2>
+        <p class="text">In the intervention, Quill activities require students to write and revise individual sentences, receiving immediate feedback that guides the revision process. Students completed 10-15 minute activities designed to fit within a variety of classroom contexts, including independent work, warm-ups, or small group rotations. Completing Quill’s practice activities in any skill area improved overall student performance by 18 percentage points, 1.6x the amount of growth demonstrated by the comparison group over the course of the school year.</p>
+      </section>
+      <img alt="Rate of growth graph" src=<%= "#{base_image_url}/growth-graph.svg" %>></img>
+    </section>
+    <section class="inner-section">
+      <section>
+        <h2 class="second">More Growth in Title I Schools</h2>
+        <p class="text">Students in Title I schools showed the greatest improvement by gaining 38 percentage points, while students in non-Title I schools gained 24 percentage points, substantially narrowing the achievement gap between the two groups.</p>
+      </section>
+      <img alt="Comparison of Title 1 and non-Title 1 schools rate of growth graph" src=<%= "#{base_image_url}/title-growth-graph.svg" %>></img>
```

Comment:
I think we need to go into more detail ("Bar graph comparing non-Title 1 schools' rate of growth (5 of 21 points gained representing a 24% improvement, with a 0.43 effect size for Cohen's d value) to Title 1 schools' ...). See examples on this page for writing alt text for graphs: https://www.section508.gov/create/alternative-text/.